packages/breadboard-cli/vite.config.ts

```diff
@@ -18,19 +18,6 @@ if (import.meta.resolve) {
 }
 
 export default defineConfig({
```

Comment:
Why is this being removed? We literally just added it in to fix an issue that was identified.

Reply:
it didn't resolve the issue. but with this pipeline it'll allow testing for potential solutions.
see here for the issue persisting in the currently deployed version:
https://github.com/ExaDev-io/breadboard/actions/runs/7817291975/job/21324831812#step:5:27